src/Components/ShowProduct.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { Dispatch, SetStateAction, useEffect, useState } from "react";
+import LandingPage from "../Pages/LandingPage";
+import { useNavigate, useParams } from "react-router-dom";
+import { Toolbar } from "primereact/toolbar";
+import { DataTable, DataTableFilterMeta } from "primereact/datatable";
+import { Dropdown, DropdownChangeEvent } from "primereact/dropdown";
+import { Column } from "primereact/column";
+import { Panel } from "primereact/panel";
+import { Fieldset } from "primereact/fieldset";
+import { FilterMatchMode } from "primereact/api";
+import { SplitButton } from "primereact/splitbutton";
+import TopContentShow from "./TopContentShow";
+import { prod } from "../data";
+
+interface Product {
```

Comment:
we can make new folder /src/models/model_name (Product here) to more organize your code.